Gemfile

```diff
@@ -27,7 +29,7 @@ gem 'gh'
 gem 'keen'
 gem 'metriks-librato_metrics'
 gem 'sentry-raven'
-gem 'simple_states'
+gem 'simple_states', git: 'https://github.com/svenfuchs/simple_states', ref: 'sf-ruby-2.4.1'
```

Comment:
Yeah running these down now. Didn't see that was happening after the merge and update. Good looking out.